pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -0,0 +1,73 @@
+//go:build policycoreutils
+// +build policycoreutils
+
+package policycoreutils
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/containers/selinuxd/pkg/semodule/seiface"
+	"github.com/go-logr/logr"
+)
+
+type SEModulePcuHandler struct {
+	logger logr.Logger
+}
+
+// Ensure that the test handler implements the Handler interface
+var _ seiface.Handler = &SEModulePcuHandler{}
+
+func runSeModule(opFlag, policyArg string) (string, error) {
+	cmd := exec.Command("/usr/sbin/semodule", opFlag, policyArg)
+	out, err := cmd.CombinedOutput()
+	return string(out), err
+}
+
+func NewSEModulePcuHandler(logger logr.Logger) (*SEModulePcuHandler, error) {
+	return &SEModulePcuHandler{logger: logger}, nil
+}
+
+func (smt *SEModulePcuHandler) SetAutoCommit(_ bool) {
+	// left to policycoreutils
+}
+
+func (smt *SEModulePcuHandler) Install(modulePath string) error {
+	out, err := runSeModule("-i", modulePath)
+	if err != nil {
+		smt.logger.Error(err, "semodule -i", "modulePath", modulePath)
+		return seiface.NewErrCannotInstallModule(modulePath)
+	}
+
+	smt.logger.Info("semodule -i", "modulePath", modulePath, "out", out)
+	return nil
+}
+
+func (smt *SEModulePcuHandler) List() ([]string, error) {
+	cmd := exec.Command("/usr/sbin/semodule", "-l")
```

Comment:
why not use the `runSeModule` function you define above?

Reply:
Thanks, that was refactoring gone wrong.